docs(plugins): basic update to splitChunks.automaticNameMaxLength docs (#3136)
webpack/webpack.js.org: src/content/plugins/split-chunks-plugin.md

By default webpack will generate names using origin and name of the chunk (e.g. `vendors~main.js`). This option lets you specify the delimiter to use for the generated names.

### `splitChunks.automaticNameMaxLength`

> `number`

Reviewer: ```suggestion
`number: 109`
```

> This option lets you specify a maximum name length to which the chunk names generated by the SplitChunksPlugin will be truncated. By default, Webpack's chunk names are truncated at 109 characters, which is on the long side and is done to prevent errors.

Reviewer: ```suggestion
Allows setting a maximum character count for chunk names that are generated by the `SplitChunksPlugin`.
```

> 

Reviewer: ```suggestion

__webpack.js.org__

```javascript
module.exports = {
  //...
  optimization: {
    splitChunks: {
      automaticNameMaxLength: 18
    }
  }
}
```

```
